src/utils/institutions-helpers.ts

```diff
@@ -26,6 +26,23 @@ export const getOrganizationHierarchy = (unit?: Organization, result: Organizati
   return getOrganizationHierarchy(unit.partOf[0], [unit, ...result]);
 };
 
+export const findAncestor = (organization: Organization): Organization => {
```

Comment:
Jeg tenker på `findAncestor` som noe som henter den som er hakket over, men ser ut som den henter den som er helt øverst? Passer det bedre med `findInstitution` (institusjon er det vi kaller det øverste nivået, Sikt, NTNU, etc, mens alt under kaller vi enhet/department/fakultet/avdeling/whatever). Eller `getTopLevelOrganization` (Organization er en fellesbetegnelse vi bruker på alle nivå, uansett om det er institusjon eller en løvnode)

Reply:
Endret til forslag nr to og fant at det allerede fantes en funksjon som het det og gjorde det samme, så nå bruker jeg den.